go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/qri-io/jsonschema
+module github.com/crossid/jsonschema
```

Comment:
most likely changed by mistake

Reply:
yeah, lets revert this bit and we can merge